doc/manuals/admin/perf_tuning.md

```diff
@@ -40,19 +40,28 @@ speeds up reads) and write efficiency (the usage of indexes slows down writes) a
 consume space in database and mapped RAM memory) and it is the administrator (not Orion) who has
 to decide what to prioritize.
 
-However, in order to help administrators in this task, the following indexes are recommended:
+However, in order to help administrators in this task, the following *general* indexes are recommended:
 
 * Collection [entities](database_model.md#entities-collection)
-    * `_id.id`
-    * `_id.type`
-    * `_id.servicePath`
-    * `attrNames`
+    * `{_id.servicePath: 1, _id.id: 1, _id.type: 1}` (note this is a compound index and key order matters in this case)
     * `creDate`
 
-In the case of using `orderBy` queries (i.e. `GET /v2/entities?orderBy=A`), it is also recommended 
+In addition, depending on your queries you could need *additional* indexes.
+
+* In the case of using attribute filters in queries (e.g. `GET /v2/entities?q=A<10`), it is recommended to create
+indexes for them. In particular if you are filtering by a given attribute 'A' then you should create an index 
+`{attrs.A.value: 1}`. If you are filtering by several attributes in the same query (e.g. `GET /v2/entities?q=A<10;B>20`
+you should combine all them in a compound index `{attrs.A.value: 1, attrs.B.value: 1}` (key order doesn't matter in this case).
+
+* In the case of using `orderBy` queries (i.e. `GET /v2/entities?orderBy=A`), it is recommended 
 to create indexes for them. In particular, if you are ordering by a given attribute 'A' in ascending order
 (i.e. `orderBy=A`) you should create an index `{attrs.A.value: 1}`. In the case of ordering by a given
-attribute 'A' in descending order (i.e. `orderBy=!A`) you should create an index `{attrs.A.value: -1}`.
+attribute 'A' in descending order (i.e. `orderBy=!A`) you should create an index `{attrs.A.value: -1}`. In the case
+of using several attributes for ordering (i.e. `orderBy=A,!B,C`) you should create a compound index taking into
+account ordering direction, i.e. `{attrs.A.value: 1, attrs.B.value: -1, attrs.C.value: 1}` (key order matters in this case)
```

Comment:
taking into￼ account **the** ordering direction

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3495/commits/693187af336cc4a95a919d0b5e8eba2807ae05fd